src/dotnet/CarbonAware.DataSources.WattTime/IRegionConverter.cs

```diff
@@ -0,0 +1,16 @@
+﻿using CarbonAware.Tools.WattTimeClient.Model;
+
+namespace CarbonAware.DataSources.WattTime;
+
+/// <summary>
+/// Represents a WattTime region converter.
```

Comment:
```suggestion
/// Represents a WattTime region converter that can translate azure region strings into the appropriate balancing authorities for WattTime.
```

Reply:
No comment, but this was updated to be a LocationConverter.